src/components/Property/RaiseWorkOrder/RaiseWorkOrderForm.js

```diff
@@ -67,30 +67,31 @@ const RaiseWorkOrderForm = ({
         ?.description
     }
 
-    if (priorityList.includes(description)) {
-      const existingCode = parseInt(
-        document.getElementById('priorityCode').value
-      )
+    if (getPriorityObjectByDescription(description)) {
       // Update priority when SOR code has priority attached if:
       // Priority description is blank, or there's only one sor code entry, or
       // when removing an SOR there's an existing entry with higher priority, or
       // the selected priority code is less than existing priority codes
       // (Higher priority as code gets lower)
+
       if (
-        !existingCode ||
+        !priorityCode ||
```

Comment:
Updating the select (or doing it behind-the-scenes via SOR code selection) now results in an update to the `priorityCode`, so we no longer need to query the DOM for it.